### PR TITLE
core-primitives: expand font and border primitives

### PR DESCRIPTION
**Pull request by ianrohde**
Summary of changes
---
- expands `font-size` options up to 72px
- adds semi-bold `font-weight` option
- adds 16px `border-radius` option

These are for additional styling options seen recently either in issues or in the core design system.

#### QA:
qa_req 0 - these are non-breaking additions

Merge checklist
---

- [ ] Bump the package version by running `npm version [patch | minor | major]` from the command line (if applicable).

> **Note:** In the context of Core Primitives, significant changes to the library or workflow, or removing primitives would be considered a major update, adding or updating primitives would be considered a minor update, and fixing primitives would be considered a patch. Non-code changes (e.g. documentation) do not require a version bump.

### CODE REVIEW

**Comment by rjmccluskey** [comment]
CR 🍻

**Comment by BaseInfinity** [comment]
very nice CR 👙

**Comment by ianrohde** [comment]
@tgines let me know if you have any requested modifications before I merge and create a new release

**Comment by ianrohde** [comment]
Updated the base `line-height` to a more accurate value after noticing that the label text was off by 1px.

**Comment by rjmccluskey** [comment]
CR 🍻

**Comment by BaseInfinity** [comment]
CR 👙

File: index.json [hunk 1]
```diff
@@ -117,6 +121,7 @@
     "sm": "2px",
     "md": "4px",
     "lg": "8px",
+    "xl": "16px",
```
Comment (BaseInfinity) [inline index.json:124]: what about `yg` for Yuge

Comment (ianrohde) [reply index.json:124]: As soon as macOS adds 32px border-radii 😜

**Review by tgines — APPROVED** [review]
Looks good to me

**Pull request merged**